docs/release-notes.rst

```diff
@@ -1,3 +1,199 @@
+
+User menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB** → **Export DB**, for exporting the F5 ACI ServiceCenter database. This option is not working on APIC versions 5.0.X and higher. This is working as expected on prior versions of the APIC. 
+
+**Workaround:** Backup the F5 ACI ServiceCenter database via the following steps:
+1. Login to the APIC server via SSH
+
+2. cd /data2/gluster/gv0/F5Networks_F5ACIServiceCenter
+
+3. Use scp, winscp or any other preferred tool to copy out the f5.db file from this location. 
+
+------
+
+**Non-default docker0 configuration on APIC may affect connection to management IP of Virtual BIG-IP**
+
+The default docker0 bridge IP has the address **172.17.0.1**. If docker0 bridge IP is updated to a non-default IP followed by APIC cluster reboot, Virtual BIG-IP login (connection to management IP) will fail from the F5 ACI ServiceCenter.
+
+**Workaround:** Change the docker0 bridge IP once more after APIC reboot to a new value. 
+
+------
+
+L2-L3 Network Management
+------------------------
+
+**Error “Invalid DN <someDn>, wrong rn prefix <somePrefix> at position X/Y“**
+
+getldevinfo.json and createbigipvlan.json APIs will show an error of the type “Invalid DN <someDn>, wrong rn prefix <somePrefix> at position X/Y“
+
+Root-cause: During VLAN creation using createbigipvlan.json API, the VLAN table in the F5 ACI ServiceCenter saves VLAN database entries. One of the fields in the VLAN table is the lifDn which is the Distinguished name of Logical Interface (in the Logical device) on APIC. If during App REST API automation, anyone creates a VLAN using createbigipvlan.json and enters invalid string in lifDn parameter of the API instead of the valid input for lifDn, the app will accept it. And on a subsequent call to getldevinfo.json or createbigipvlan.json throw the aformentioned error.
+
+**Workaround**: Uninstall and re-install the application to clean out the F5 ACI ServiceCenter database.
+
+------
+
+
+L4-L7 App Services
+------------------
+
+**Application services declaration not deleted**
+
+If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, the Application Services configuration may not be deleted completely, even after multiple attempts from the application UI. However, the configuration gets removed from the BIG-IP device.
+
+**Workaround**: Upload one more AS3 sample declaration to the app and then perform a :guilabel:`Delete all` operation. (Use :guilabel:`View AS3 Declaration` and click :guilabel:`Delete`.)
+
+-------
+
+
+Dynamic Endpoint Attach Detach
+------------------------------
+
+**Dynamic Endpoint attach/detach is not supported for BIG-IP High Availability setups**
+
+Dynamic Endpoint attach/detach using the "Manage Endpoint Mappings" button is not supported for BIG-IP devices which are in HA pair. If used, the behavior is unknown and user may experience BIG-IP service restarts and hang up.
```

Comment:
..which are in an HA pair.
..and users may experience BIG-IP service restarts and hang ups.